content/en/blog/_posts/2024-08-15-moving-cgroup-v1-support-maintenance-mode-kubernetes-1-31.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: blog
+title: "Kubernetes 1.31: Moving cgroup v1 Support into Maintenance Mode"
+date: 2024-08-15
```

Comment:
Is this publication date correct? Also, do we want to specify a _time_?

I would, to ensure this is ordered after the release announcement article (which also needs a specific publication time set).

It's OK to merge and publish the article slightly after the publication time, so long as it's not many hours later (< 20 hours later feels OK).

Reply:
@@rashansmith what do you think about the publishing date?

xref: kubernetes/enhancements#4569 (comment)

Reply:
@harche your publication date from our blog project board is set on the 14th of August, which would be 24 hours after the release I think you should adjust according to that.

@rashansmith correct me if I'm wrong here!

@sftim what happens if you don't specify a time? Does it go off at midnight 0:00 UTC?

Reply:
No time specified? The article comes out at midnight in the default timezone.

Reply:
I think it would be ok to publish potentially in the 24-hour span after the release, wdyt? @sftim

Reply:
> I think it would be ok to publish potentially in the 24-hour span after the release, wdyt? @sftim

No objection from me (as a blog team editor). Release comms are the group that get to decide on scheduling here.

Reply:
Thanks @mbianchidev and @sftim, I updated the date to `2024-08-14`